src/traffic_layout/engine.cc

```diff
@@ -59,8 +60,10 @@ static bool fix_flag = false;
 std::string
 check_path(const std::string &path)
 {
+  std::error_code ec;
   std::string yaml_file = Layout::relative_to(path, "runroot.yaml");
```

Comment:
I'd probably not have the `std::string` at all, e.g.
```
ts::file::path yaml_fs { Layout::relative_to(path, "runroot.yaml") };
```
Then `yaml_file.c_str()` because `yaml_fs.c_str()`.